verilog/tools/kythe/kythe_facts_extractor.h

```diff
@@ -229,6 +229,9 @@ class KytheFactsExtractor {
   void GenerateEdgeString(const VName& source, absl::string_view name,
                           const VName& target) const;
 
+  // Create the signature for the given name (variable, module ..etc).
+  const std::string CreateSignature(absl::string_view name) const;
```

Comment:
The return type doesn't need `const` (unless it is a reference).

Reply:
Since this function doesn't use any class members, it should be a `static` method.